plugins/default.nix

```diff
@@ -12,8 +12,8 @@
     ./colorschemes/everforest.nix
     ./colorschemes/github-theme.nix
     ./colorschemes/gruvbox.nix
-    ./colorschemes/kanagawa.nix
     ./colorschemes/kanagawa-paper.nix
+    ./colorschemes/kanagawa.nix
```

Comment:
This looks like one author has sorted with one algorith, then you've sorted with another.

I don't see any value in keeping this change, since it could easily flip back and forth.

If we wanted to do this kinda cleanup, I'd prefer it be done by adding formatters or linters to treefmt.